flow-typed/box-ui-elements.js

```diff
@@ -618,6 +618,111 @@ type Tasks = {
     total_count: number,
 };
 
+/* New Task Types START */
+
+type ID = string;
+type ISODate = string;
+
+type UserMini = {
```

Comment:
Was there a reason you decided to created a new type, instead of using `User`? They appear to be identical

Reply:
I guess they are... I can only guarantee accuracy for the Tasks API though. Can alias for now?